### PR TITLE
Refactor/clean up code

### PR DESCRIPTION
**Pull request by Sosodope**

### CODE REVIEW

File: client/src/components/CreateGroup/createGroupMutation.graphql [hunk 1]
```diff
@@ -2,6 +2,7 @@ mutation CreateGroup($input: GroupInput!){
   createGroup(input: $input) {
     id
     name
-    description
+    description,
```
Comment (ivanms1) [inline client/src/components/CreateGroup/createGroupMutation.graphql]:
```suggestion
    description
```

File: client/src/components/GroupHeader/GroupHeaderContainer.js [hunk 2]
```diff
@@ -42,11 +17,25 @@ export default class GroupHeaderContainer extends Component {
...
 }
 	render() {
+		const { id } = this.props.match.params
+
 		return (
-			<div className="group-header-container"> 
-				<GroupHeaderBanner />
-				<GroupHeaderMenu member={this.state.member}/>
-			</div>
+			<Query query={GET_GROUP} variables={{ id }}>
```
Comment (ivanms1) [inline client/src/components/GroupHeader/GroupHeaderContainer.js]: I think your tab size is too big, it makes this hard to read sometimes.

**Review by ivanms1 — APPROVED** [review]

File: client/src/components/CreateGroup/createGroupMutation.graphql (hunk 1 above)
Comment (Sosodope) [reply client/src/components/CreateGroup/createGroupMutation.graphql]: thanks for catching this

File: client/src/components/GroupHeader/GroupHeaderContainer.js (hunk 2 above)
Comment (Sosodope) [reply client/src/components/GroupHeader/GroupHeaderContainer.js]: sorry about that, i'll switch to using spaces now

**Pull request merged**